/**:
  ros__parameters:

    error_coeff: 0.0002

    limit_view_angle: 65.0

    obstacle_radius: 0.3

    covariance:
      [
        0.0225, 0.0,   0.0,   0.0,      0.0,      0.0,
        0.0,   0.0225, 0.0,   0.0,      0.0,      0.0,
        0.0,   0.0,   0.0225, 0.0,      0.0,      0.0,
        0.0,   0.0,   0.0,   0.000625, 0.0,      0.0,
        0.0,   0.0,   0.0,   0.0,      0.000625, 0.0,
        0.0,   0.0,   0.0,   0.0,      0.0,      0.000625,
      ]
